locations/spiders/sok_tr.py

```diff
@@ -0,0 +1,97 @@
+import re
+from typing import Iterable
+
+import scrapy
+
+from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
+from locations.pipelines.address_clean_up import clean_address, merge_address_lines
+
+PROVINCES_URL = "https://kurumsal.sokmarket.com.tr/ajax/servis/sehirler"
+DISTRICTS_URL = "https://kurumsal.sokmarket.com.tr/ajax/servis/ilceler?city={province}"
+STORES_URL = "https://kurumsal.sokmarket.com.tr/ajax/servis/magazalarimiz?city={province}&district={district}"
+
+
+class SokTRSpider(scrapy.Spider):
+    name = "sok_tr"
+    item_attributes = {"brand": "ŞOK", "brand_wikidata": "Q19613992"}
```

Comment:
NSI has this as `Şok`

https://nsi.guide/index.html?t=brands&k=shop&v=supermarket&tt=q19613992

Reply:
I looked at the website and the naming is not consistent. They sometimes use the name with all caps when referring to the chain stores ("ŞOK marketleri") or sometimes with normal capitalization. I can change it to "Şok" if it is more appropriate.

Pages I've looked at:
https://kurumsal.sokmarket.com.tr/tarihcemiz
https://kurumsal.sokmarket.com.tr/insan-kaynaklari/insan-kaynaklari-politikamiz
https://kurumsal.sokmarket.com.tr/markalar/soka-ozel-urunler

Reply:
I have a strong preference being the same as NSI. But if NSI is wrong, it can be changed there.